### PR TITLE
CDAP-12323 Remove authorization bootstrapper since it is no longer needed for CDAP to starup

### PR DESCRIPTION
**Pull request by poornachandra**
JIRA - https://issues.cask.co/browse/CDAP-12323
Build - https://builds.cask.co/browse/CDAP-RUT1246-4

### CODE REVIEW

File: cdap-security/src/main/java/co/cask/cdap/security/authorization/AuthorizationBootstrapper.java [hunk 1]
```diff
@@ -1,109 +0,0 @@
...
-    String currentUser;
-    try {
-      currentUser = UserGroupInformation.getCurrentUser().getShortUserName();
-    } catch (IOException e) {
-      throw Throwables.propagate(e);
-    }
-    this.systemUser = new Principal(currentUser, Principal.PrincipalType.USER);
-    this.adminUsers = getAdminUsers(cConf);
-    if (enabled && adminUsers.isEmpty()) {
-      LOG.info("Admin users specified by {} is empty.", Constants.Security.Authorization.ADMIN_USERS);
```
Comment (yaojiefeng) [inline cdap-security/src/main/java/co/cask/cdap/security/authorization/AuthorizationBootstrapper.java:65]: Do we need to remove this constant from cdap-site? I dont think it is useful any more

**Review by yaojiefeng — COMMENTED** [review]
Just one comment

File: cdap-security/src/main/java/co/cask/cdap/security/authorization/AuthorizationBootstrapper.java (hunk 1 above)
Comment (poornachandra) [reply cdap-security/src/main/java/co/cask/cdap/security/authorization/AuthorizationBootstrapper.java:65]: Good point

**Review by yaojiefeng — APPROVED** [review]
LGTM

**Comment by poornachandra** [comment]
Build is green, merging.

**Pull request merged**